tutorials/cloud-iot-rtdp/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Real time Data Processing With Google Cloud IoT
+description: Demonstrate how to build data processing backend for Google Cloud IoT.
+author: teppeiy
+tags: Cloud IoT Core, Cloud Dataflow, Cloud Functions, Java, Node.js
+date_published: 2017-11-29
+---
+# Real time Data Processing With Google Cloud IoT
+This repository contains sample code for Real time Data Processing With Google Cloud IoT solution. Sample MQTT client generates simulated data and submits them to Cloud IoT Core. Cloud Functions and Cloud Dataflow receive data through Cloud Pub/Sub.
```

Comment:
Can you expand this intro to include some context about the various technologies mentioned? And maybe a more detailed context for the sample's functionality? "Real time data processing" is pretty vague. What processing does this example actually do?

Reply:
Will add more context and explanation of what this sample code does.